### ISSUE TITLE
[suggestion] Submit to F-droid

### ISSUE THREAD

**Issue by EdoSecco** [AzeemIdrisi/InstaLoaderApp#1]
I think it would be great to have this app available also on [F-droid](https://f-droid.org/).

**Comment by AzeemIdrisi** [comment]
Yeah you're right, I will try to publish it on F-Droid. Thank You.

**Comment by EdoSecco** [comment]
Hi, still not present there?

**Comment by AzeemIdrisi** [comment]
F-droid's submission process has a lot of friction, hence dropped.

**Comment by EdoSecco** [comment]
Ah, yes, I know 🙄
But you can ask to have the APK added to another Fdroid-like repo that has easier policy: https://apt.izzysoft.de/fdroid/
